Build on Linux?

Hi Kai

Any chance to build OctaChainer on Linux?

**EDIT**: I try to build it on a Ubuntu 17.10 machine. I installed the dependencies and changed the OctaChainer.pro file, from:

    macx{
    TARGET = OctaChainerv1_3_OSX
    }
    else
    {
    TARGET = OctaChainer
    }

to:

    macx {
    TARGET = OctaChainerv1_3_OSX
    } else {
    TARGET = OctaChainer
    }

`qmake` works, no errors, but unfortunately i get the following error when trying to build:

    $ make
	g++ -m64 -Wl,-O1 -o OctaChainer main.o mainwindow.o audiofactory.o otwriter.o tabmainwidget.o tabmegabreakwidget.o tabgridwidget.o tabstepswidget.o projectsettings.o moc_mainwindow.o moc_audiofactory.o moc_tabmainwidget.o moc_tabmegabreakwidget.o moc_tabgridwidget.o moc_tabstepswidget.o   -L/usr/X11R6/lib64 -lQt5Multimedia -lQt5Widgets -lQt5Gui -lQt5Network -lQt5Core -lGL -lpthread 
	/usr/bin/ld: audiofactory.o: undefined reference to symbol 'sf_read_int@@libsndfile.so.1.0'
	//usr/lib/x86_64-linux-gnu/libsndfile.so.1: error adding symbols: DSO missing from command line
	collect2: error: ld returned 1 exit status
	Makefile:182: recipe for target 'OctaChainer' failed
	make: *** [OctaChainer] Error 1

Any hints?

Thank you.

Thank you @Wedmer for the help, unfortunately the error message is the same.

    $ make
    g++ -m64 -Wl,-O1 -o OctaChainer main.o mainwindow.o audiofactory.o otwriter.o tabmainwidget.o tabmegabreakwidget.o tabgridwidget.o tabstepswidget.o projectsettings.o moc_mainwindow.o moc_audiofactory.o moc_tabmainwidget.o moc_tabmegabreakwidget.o moc_tabgridwidget.o moc_tabstepswidget.o   -L/usr/X11R6/lib64 -lQt5Multimedia -lQt5Widgets -lQt5Gui -lQt5Network -lQt5Core -lGL -lpthread
    /usr/bin/ld: audiofactory.o: undefined reference to symbol 'sf_read_int@@libsndfile.so.1.0'
    //usr/lib/x86_64-linux-gnu/libsndfile.so.1: error adding symbols: DSO missing from command line
    collect2: error: ld returned 1 exit status
    Makefile:181: recipe for target 'OctaChainer' failed
    make: *** [OctaChainer] Error 1

Is the double slash on `//usr/lib/x86_64-linux-gnu/libsndfile.so.1` correct?

--- a/OctaChainer.pro
+++ b/OctaChainer.pro
@@ -49,7 +49,7 @@ macx{
     INCLUDEPATH += $$PWD/libsndfile/include
     DEPENDPATH += $$PWD/libsndfile/include
 }else{
-    *-g++{
+    *-g++*{
         LIBS+=-lsndfile
     }
 }

